From 78c0e2b06a65b5d3fb66875f63ef0402127d34e2 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Sat, 13 Jun 2020 18:39:42 -0700
Subject: [PATCH] build: do not descend into po

OpenWrt does not install motion's translations, and building them pulls
in the full gettext tooling for no gain.

Upstream-Status: Inappropriate [OpenWrt build configuration]
---
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 
-SUBDIRS = src po
+SUBDIRS = src
 
 pkgsysconfdir = $(sysconfdir)/@PACKAGE@
 dist_pkgsysconf_DATA = \
--- a/configure.ac
+++ b/configure.ac
@@ -448,7 +448,6 @@ AC_SUBST(AM_LDFLAGS, "$TEMP_LDFLAGS")
 AC_CONFIG_FILES([
 	Makefile
 	src/Makefile
-  po/Makefile.in
   data/camera1-dist.conf
   data/camera2-dist.conf
   data/camera3-dist.conf
